### ISSUE TITLE
Repeate message with newlines for a better overview

### ISSUE THREAD

**Issue by joschi36** [claranet/nagitheus#6]
Hey,

Thanks for your great work. We are currently using your script for Business Process checks inside of Icinga.
I have a suggestion and want to know what you guys think:

Repeating the message without newlines for a better overview when many alerts hit. So the output would be like this:

**Command:**
```
go run nagitheus.go -H 'https://prometheus.mgt.domain.com' -q "(kubelet_volume_stats_used_bytes/kubelet_volume_stats_capacity_bytes*100)>2" -w 2  -c 5  -m ge -u UN -p PW -l persistentvolumeclaim
```

**Output:**
```
WARNING prometheus-kube-prometheus-db-prometheus-kube-prometheus-0 is 2.2607424766047886 CRITICAL prometheus-kube-prometheus-db-prometheus-kube-prometheus-0 is 5.625835543270624

WARNING prometheus-kube-prometheus-db-prometheus-kube-prometheus-0 is 2.2607424766047886 
CRITICAL prometheus-kube-prometheus-db-prometheus-kube-prometheus-0 is 5.625835543270624
```

**Comment by domcar** [comment]
Hello @joschi36 , thanks a lot for the Issues that you opened. It looks definitely better with a new line.
Unfortunately I don't know how much time i have to address this Issue (and the others).
Any chance that you want to make a PR?

**Comment by ramedlawH** [comment]
Multiline output is added with #12  We released the new version 1.4.2 where you can use the `--print-details` flag to show a multiline detailed output.
